tests/plugins/widget/manual/contextmenushortcuts.html

```diff
@@ -0,0 +1,10 @@
+<div id="editor">
+	<p><span class="math-tex">\(x = {-b \pm \sqrt{b^2-4ac} \over 2a}\)</span></p>
+</div>
+
+<script>
+	CKEDITOR.replace( 'editor', {
+		extraPlugins: 'mathjax',
```

Comment:
It may be better to stick into a bit simple widget like `placeholder`. You will avoid additional configuration and the test will load faster. Also, include plugin by bender tags like it was done for `contextmenu`.